src/main/java/ui/Menu.java

```diff
@@ -0,0 +1,27 @@
+package ui;
+import java.util.Scanner;
+
+public class Menu {
```

Comment:
This class is started too look more like UserInput rather than Menu for me. Maybe it is a good place to store parts of the Menu code?

Reply:
To be confirmed